src/Middleware/AuthTokenMiddleware.php

```diff
@@ -123,4 +131,11 @@ private function fetchToken()
             return $auth_tokens['access_token'];
         }
     }
+
+    private function getQuotaProject()
+    {
+        if ($this->fetcher instanceof GetQuotaProjectInterface) {
```

Comment:
Are there any cases where things get wrapped and this interface would get lost?

e.g. 
```
$urc instanceof UserRefreshCredentials // true
$cred instanceof GetQuotaProjectInterface // true
$cred = cacheWrapper($urc)
$cred instanceof GetQuotaProjectInterface // false
```

Reply:
Good catch! Yes, I absolutely need to add this to the cache wrapper